### PR TITLE
New package: stig-0.10.0a

### PR DESCRIPTION
**Pull request by autrimpo**
As [transmission-remote-cli](https://github.com/fagga/transmission-remote-cli) is no longer maintained, I've packaged another terminal client for transmission.

### CODE REVIEW

File: srcpkgs/python-natsort/template [hunk 1]
```diff
@@ -0,0 +1,32 @@
+# Template file for 'python-natsort'
+pkgname=python-natsort
+version=5.3.3
+revision=2
```
Comment (maxice8) [inline srcpkgs/python-natsort/template]: revision=1

File: srcpkgs/python-natsort/template [hunk 2]
```diff
@@ -0,0 +1,32 @@
...
+noarch=yes
+wrksrc=natsort-${version}
```
Comment (maxice8) [inline srcpkgs/python-natsort/template]: quote

File: srcpkgs/python-natsort/template [hunk 3]
```diff
@@ -0,0 +1,32 @@
...
+build_style=python-module
```
Comment (maxice8) [inline srcpkgs/python-natsort/template]: move pycompile_module= to directly below build_style=

File: srcpkgs/python-natsort/template [hunk 4]
```diff
@@ -0,0 +1,32 @@
...
+hostmakedepends="python-setuptools python3-setuptools"
+makedepends="python-devel python3-devel"
```
Comment (maxice8) [inline srcpkgs/python-natsort/template]: makedepends= doesn't seem necessary here

File: srcpkgs/stig/template [hunk 5]
```diff
@@ -0,0 +1,20 @@
...
+revision=2
+noarch=yes
+build_style=python3-module
+hostmakedepends="python3-setuptools"
+makedepends="python3-devel"
+depends="python3 python3-aiohttp python3-blinker python3-natsort python-urwid python3-urwidtrees python3-xdg"
+pycompile_module="stig"
+short_desc="TUI and CLI for the BitTorrent client Transmission"
+maintainer="Michal Koutenský <koutak.m@gmail.com>"
+license="GPL-3.0"
```
Comment (maxice8) [inline srcpkgs/stig/template]: Please use SPDX identifier, it is missing -or-later or -only

File: srcpkgs/stig/template [hunk 6]
```diff
@@ -0,0 +1,20 @@
+# Template file for 'stig'
+pkgname=stig
+version=0.9.0a
+revision=2
```
Comment (maxice8) [inline srcpkgs/stig/template]: revision=1

**Comment by maxice8** [comment]
Update check for them is broken

**Comment by maxice8** [comment]
Stig update check can be fixed with 
```
site=https://github.com/rndusr/stig/releases
pkgname=v
```

in the `update` file

File: srcpkgs/stig/template [hunk 7]
```diff
@@ -0,0 +1,20 @@
...
+revision=1
+noarch=yes
+build_style=python3-module
+hostmakedepends="python3-setuptools"
+makedepends="python3-devel"
+depends="python3 python3-aiohttp python3-blinker python3-natsort python-urwid python3-urwidtrees python3-xdg"
+pycompile_module="stig"
+short_desc="TUI and CLI for the BitTorrent client Transmission"
+maintainer="Michal Koutenský <koutak.m@gmail.com>"
+license="GPL-3.0-only"
```
Comment (maxice8) [inline srcpkgs/stig/template]: No, the license field says

```
 license = 'GPLv3+',
```

which is -or-later

File: srcpkgs/stig/template [hunk 8]
```diff
@@ -0,0 +1,20 @@
+# Template file for 'stig'
+pkgname=stig
+version=0.9.0a
+revision=1
+noarch=yes
+build_style=python3-module
+hostmakedepends="python3-setuptools"
+makedepends="python3-devel"
+depends="python3 python3-aiohttp python3-blinker python3-natsort python-urwid python3-urwidtrees python3-xdg"
```
Comment (maxice8) [inline srcpkgs/stig/template]: fold this at 80 columns.

you can use `fold -w 80 -s` to get it folded, add a whitespace on the new line.

File: srcpkgs/stig/template (hunk 7 above)
Comment (autrimpo) [reply srcpkgs/stig/template]: ah sorry, my bad

**Comment by autrimpo** [comment]
Updated python-natsort to 5.4.0 which was released today

**Comment by autrimpo** [comment]
Updated stig to 0.10.0a as well

File: srcpkgs/stig/template [hunk 9]
```diff
@@ -0,0 +1,21 @@
+# Template file for 'stig'
+pkgname=stig
+version=0.10.0a
+revision=1
+noarch=yes
+build_style=python3-module
+hostmakedepends="python3-setuptools"
+makedepends="python3-devel"
+depends="python3 python3-aiohttp python3-blinker python3-natsort python-urwid
```
Comment (maxice8) [inline srcpkgs/stig/template]: python3-urwid you mean ?

doesn't make sense for a python3 package to depend on python2

Comment (autrimpo) [reply srcpkgs/stig/template]: Yes, apologies.

**Pull request merged**